ember-file-upload/package.json

```diff
@@ -46,15 +46,10 @@
     "prepare": "pnpm build"
   },
   "dependencies": {
-    "@ember/test-helpers": "^2.9.3",
     "@ember/test-waiters": "^3.0.0",
     "@embroider/addon-shim": "^1.5.0",
     "@embroider/macros": "^1.0.0",
-    "@glimmer/component": "^1.0.4",
-    "@glimmer/tracking": "^1.0.4",
-    "ember-auto-import": "^2.0.0",
-    "ember-modifier": "^3.2.7 || ^4.0.0",
-    "tracked-built-ins": "^3.0.0"
+    "ember-auto-import": "^2.0.0"
   },
```

Comment:
Believe the remaining dependencies are required. Keen for others to check this, though.

Reply:
Looks good to.